src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -170,8 +170,8 @@
     <Compile Include="Interfaces\ILibraryViewCustomization.cs" />
     <Compile Include="Interfaces\LayoutSpecification.cs" />
     <Compile Include="Properties\Resources.Designer.cs">
-      <AutoGen>True</AutoGen>
       <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
```

Comment:
Is this change useful in any way? Looks like this change can be reverted.

Reply:
it was moved here automatically ....I think
I can move it back